src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -545,6 +547,10 @@ def transform_request(self,
         if entity_type == 'projects':  # Add project summaries to each project hit
             self.add_project_summaries(final_response['hits'], es_response)
 
+        if include_file_urls:
+            for h in final_response['hits']:
+                for data_file in h['files']:
+                    data_file['url'] = f"{config.dss_endpoint}/files/{data_file['uuid']}?replica=aws&version={data_file['version']}"
```

Comment:
This line looks like it's too long. If so, please wrap but without using backslash.